### DISCUSSION TITLE
Need help with two things.

### DISCUSSION THREAD

**Discussion by AxzonZero** [dialogic-godot/dialogic#1594] [Q&A]
Let me just start off by saying love Dialogic, it's really helpful.
That being said, I don't know if what's going wrong is just a bug or something wrong that I did.
I've imported a folder of character portraits. But they never show up when I want them to. Dialogic says that they are detected, yet they never show up when the game is running.

The other thing I need help may not be related to Dialogic itself.
Upon pressing enter for the last time and having the last dialogue disappear, how do I make it so that an animation plays that I have set up using my AnimationPlayer node?

**Comment by Jowan-Spooner** [comment]
Hey glad you like dialogic. Some more information would be very useful: 
- Are you using dialogic 1 (godot 3.x) or dialogic 2 (alpha, godot 4.x)?
- What do you mean the portraits never show up? Are you joining the characters?

For the animation you can listen to a signal that is emitted when the dialog ends:
- Dialogic.timeline_ended for dialogic 2
and 
- dialog.timeline_end for dialogic 1 (this signal is on the node returned from Dialogic.start() and the function has to take an argument)

**Reply by AxzonZero** [reply]
Thanks for trying to help. I think I'm using Dialogic 1 (I'm using Godot 3.5.1).
When I mean the portraits never show up, it's exactly what it sounds like. I have a timeline called "Intro." I imported a folder of .png files, and linked them to a character named Brandon. When I run the game, they are not visible.
![Char](https://github.com/coppolaemilio/dialogic/assets/55718674/12fbd65a-543b-430c-9d8b-3615d85475d8)
![Screenshot 2023-06-03 121534](https://github.com/coppolaemilio/dialogic/assets/55718674/b57a3285-2d06-485a-84e2-44bac1c4b5f8)
![Screenshot 2023-06-03 121517](https://github.com/coppolaemilio/dialogic/assets/55718674/ef70587b-f096-4c42-9435-96d3b06f8540)

**Reply by Jowan-Spooner** [reply]
As I said, portraits only appear if the character is joined (with the character event). In your timeline he is not considered to be "there". When joining you can also specify the position the charcter should be in.

**Reply by AxzonZero** [reply]
So, it looks like I'm back--I'm having trouble with the animation part. How do I emit the signal dialog.timeline_end?
